app/controllers/chargeback_assignment_controller.rb

```diff
@@ -4,110 +4,79 @@ class ChargebackAssignmentController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
-  include Mixins::SavedReportPaging
   include Mixins::GenericSessionMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
     @table_name ||= "chargeback_assignment"
   end
 
-  def tree_select
-    self.x_active_tree = params[:tree] if params[:tree]
-    self.x_node = params[:id]
-    get_node_info(x_node)
-    replace_right_cell
+  def index
+    @title = _("Chargeback Assignments")
+    @tabform = "Compute"
+    session[:changed] = @changed = false
+    build_tabs
+    set_form_vars
+    render :action => "index"
```

Comment:
You probably don't need this line here, the name of the methoud should implicitly match the view.